example/src/Home.tsx

```diff
@@ -0,0 +1,160 @@
+import { Typo } from '@components/Typo';
+import * as Membrane from '@jellyfish-dev/react-native-membrane-webrtc';
+import React, { useCallback, useContext, useEffect } from 'react';
+import {
+  View,
+  Text,
+  Pressable,
+  TextInput,
+  StyleSheet,
+  Platform,
+  PermissionsAndroid,
+  Switch,
+  Alert,
+} from 'react-native';
+
+import { VideoroomContext } from './VideoroomContext';
+
+export const Home = ({ navigation }) => {
+  const videoroomContext = useContext(VideoroomContext);
+  const [roomName, setRoomName] = videoroomContext!.roomNameState;
```

Comment:
Why do we need `!` here? Have you looked into the link I attached in the task description? It has a solution to exactly this problem: https://kentcdodds.com/blog/how-to-use-react-context-effectively#typescript

Reply:
Yes I read it, I just thought it would look silly with the amount of values I wanted to store there.  After other changes I think it makes sens now. Done.